src/main/java/me/youhavetrouble/purpurextras/config/PurpurConfig.java

```diff
@@ -27,6 +27,7 @@ public class PurpurConfig {
     public final boolean upgradeStoneToIronTools;
     public final boolean upgradeIronToDiamondTools;
     public final String beeHiveLoreBees, beeHiveLoreHoney;
+    public final double netherBuildBlockHeight;
```

Comment:
unnecessary double. Integer would do as blocks are on a fixed grid with distance of 1 anyway